README.md

```diff
@@ -28,8 +28,19 @@ conda_build:
 CONDA_BUILD_SYSROOT:
   - /opt/MacOSX10.10.sdk # [osx]
 ```
-5. Use `conda build recipes --croot=<output dir>` to build the `dalphaball`
-   conda package.
+5. Create and activate a new conda environment (or activate an existing conda environment), 
+installing `conda-build` and `pyrosetta` packages. For older versions of conda (<=4.6), one may need to install
+`conda-build<=3.22.0` to avoid an `ImportError` at the next stage.
+6. Navigate to this cloned repository directory, and use `conda build recipes -c conda-forge --croot=<output dir>`
+to build the `dalphaball` conda package.
+7. In order to use the `dalphaball` package with `pyrosetta`, then use `conda install dalphaball -c <output dir>`
+to install `dalphaball` into the active conda environment. Optionally, it may be installed into any conda environment
+at this stage. 
+8. The `dalphaball` executable is then located in the conda environment `bin` directory, usually located here:
```

Comment:
If you're in the conda environment, the MYENV/bin should be in $PATH - I think you can simplify number 10 to `-holes:dalphaball dalphaball`

Reply:
@klimaj - can you test this and we'll get this updated and merged asap!

Reply:
However, with the conda environment active, on command line `echo $PATH` shows the correct `/bin` directory.

Reply:
I think I figured it out  - I had launched `jupyter notebook` from a different conda environment, even though I set the kernel to the new conda environment. Hence `$PATH` was coming from the conda environment running Jupyter notebooks, not from the kernel.

Reply:
@weitzner I've updated step 9 to reflect your suggestion

Reply:
If you're using a different env for jupyter, could you/should you install the `ipykernel` package to make the environment available as a kernel type?

Reply:
Yes - that's a good point! One would need to install `ipykernel`, but that is already a requirement when installing `jupyter`. However, I think this is more of an edge case since I already had `jupyter notebook` running, and used `ipykernel` to register the new conda environment as a kernel. I think users would most of the time launch jupyter from the same environment as the kernel. In this case, step 9 is a bit verbose (we could just document the standard case).

Reply:
I think you have to install `ipykernel` in each environment you want to expose to the jupyter server!

Reply:
Got it! Yes that makes sense, at least `ipykernel` should be installed, and the environment registered. I just updated step 9 to note this.